libcore/src/IO/IniFileParser.cpp

```diff
@@ -235,12 +235,10 @@ bool IniFileParser::ParseHeader(TiXmlNode * xHeader)
 
         std::string format = xHeader->FirstChildElement("trajectories")->Attribute("format") ?
                                  xHeader->FirstChildElement("trajectories")->Attribute("format") :
-                                 "xml-plain";
+                                 "plain";
         std::transform(format.begin(), format.end(), format.begin(), ::tolower);
 
-        if(format == "xml-plain") {
-            _config->SetFileFormat(FileFormat::XML);
-        } else if(format == "plain") {
+        if(format == "plain") {
```

Comment:
Maybe remove the option `format` from the inifile?

Reply:
I would keep this, because it also emits a warning if the format is not set or wrong.